deny.toml

```diff
@@ -0,0 +1,263 @@
+# This template contains all of the possible sections and their default values
```

Comment:
This file is really long 😆 

Its basically the empty one the tool generates, and I'll comment on the one place Ive made changes